simcore/simulation/library/anchor.cpp

```diff
@@ -33,7 +34,12 @@ double const Anchor::GetBondLambda() { return bond_lambda_; }
 void Anchor::SetBondLambda(double l) { bond_lambda_ = l; }
 void Anchor::SetMeshLambda(double ml) { mesh_lambda_ = ml; }
 
-void Anchor::SetDiffusion() { kick_amplitude = sqrt(24.0 / (diameter_*delta_)); }
+void Anchor::SetDiffusion() {
+  // Solve them now so you do not have to keep taking sqrts
+  kick_amp_s_ = sqrt(2. * diffusion_s_);
```

Comment:
Where are diffusion_s_ and diffusion_d_ being set in anchor? I'm not finding it. Also, shouldn't there be a factor of delta in the square root to get the right units when we have `dr = kick_amp_s_ * normal(1) * delta`?

Reply:
You were correct on both counts. I checked the velocity functions but didn't rigorously check the diffusion step because it got late. That was sloppy of me.

kick_amp_s/d_ = sqrt(2* diffusion_s/d_*delta_)

The correct update should be `dr = kick_amp_s/d_ * normal(1)`. No delta required during this portion of the code.

Corrections have been pushed.